snapshots/snapshots/src/main/kotlin/com/emergetools/snapshots/SnapshotSaver.kt

```diff
@@ -93,15 +98,21 @@ internal object SnapshotSaver {
     outputFile.outputStream().use { writer(it) }
   }
 
-  private fun safeFilename(userDefinedName: String): String {
+  /**
+   * Normalize the user defined name to a key name that can be used as a filename and for
+   * later indexing when comparing screenshots.
+   */
+  private fun keyName(userDefinedName: String): String {
```

Comment:
nit: I did like the old `safe` naming which implies the the regex replacement.

Reply:
I think key fits a bit better given it's used for both the key we diff with as well as filename, so will leave as-is.